remme/clients/account.py

```diff
@@ -56,21 +56,21 @@ def transfer(self, address_to, value):
 
         return self._send_transaction(AccountMethod.TRANSFER, transfer, addresses_input, addresses_output)
 
-    def get_account(self, address):
+    async def get_account(self, address):
         account = Account()
-        account.ParseFromString(self.get_value(address))
+        account.ParseFromString(await self.get_value(address))
```

Comment:
From the readability point of view it is better to assign `await` statements to separate variables like `raw_account`.